src/lib/systemjs-angular-loader.js

```diff
@@ -0,0 +1,49 @@
+var templateUrlRegex = /templateUrl\s*:(\s*['"`](.*?)['"`]\s*)/gm;
```

Comment:
it is needed for testing. Karma uses systemjs to run the tests so it needs either `module.id` or the loader.

I tried to use the loader at `demo/` but I couldn't. It needed to be on SystemJS baseURL :/